pkg/virt-launcher/virtwrap/converter/archconverter.go

```diff
@@ -43,14 +43,14 @@ type ArchConverter interface {
 }
 
 func NewArchConverter(arch string) ArchConverter {
-	switch {
-	case isARM64(arch):
+	switch arch {
+	case "arm64":
 		return archConverterARM64{}
-	case isPPC64(arch):
+	case "ppc64le":
 		return archConverterPPC64{}
-	case isS390X(arch):
+	case "s390x":
 		return archConverterS390X{}
-	case isAMD64(arch):
+	case "amd64":
```

Comment:
I think we should add constants for these. wdyt?

Reply:
Done!

Reply:
Awesome :) thanks!
if you don't mind using them in the test code as well it could be great :)

Reply:
Done :)